Project.toml

```
@@ -19,5 +19,4 @@ Printf = "1"
 Roots = "2.1.6"
 Static = "1.1.1"
 StaticArrays = "1.9"
-Trixi = "0.11.10"
-julia = "1.10"
+julia = "1.10"
```

```suggestion
Trixi = "0.11.10"
julia = "1.10"
```
I would not remove the compat bound on Trixi.jl. One should always have compat bounds for all dependencies.

---

This compat bound will be put back when we remove the workaround to get the CI to run

---

```suggestion
Trixi = "0.12"
julia = "1.10"
```

This is what we will need